docs/user-guide/programmatic-usage.md

```diff
@@ -24,4 +24,4 @@ See https://github.com/DataDog/stratus-red-team/tree/main/examples
 
 ## Reference
 
-https://pkg.go.dev/github.com/datadog/stratus-red-team/pkg/stratus
+https://pkg.go.dev/github.com/datadog/stratus-red-team/v2/pkgstratus
```

Comment:
This should be `https://pkg.go.dev/github.com/datadog/stratus-red-team/v2/pkg/stratus`. `https://pkg.go.dev/github.com/datadog/stratus-red-team/v2/pkgstratus` returns 404